### PR TITLE
[Tech Radar] Add Relay, Redux, and Fastlane

### PR DESCRIPTION
**Pull request by alloy**
https://github.com/artsy/README/pull/198#issuecomment-489619450

### CODE REVIEW

File: playbooks/technology_radar/data.csv [hunk 1]
```diff
@@ -46,3 +46,6 @@ APIs as a product,assess,techniques,TRUE,
 GraphQL batching,assess,techniques,TRUE,
 Bug bounties,hold,techniques,FALSE,
 Serverless architecture,hold,techniques,TRUE,
+Relay,adopt,languages & frameworks,FALSE,
+Redux,assess,languages & frameworks,FALSE,
```
Comment (alloy) [inline playbooks/technology_radar/data.csv]: @damassi While we’re using this in production (eg Prediction), with the newer React hooks API I _think_ the understanding is that in the future we still need to assess if we ever need to add full Redux again or if the builtin API is enough (like has been done in the new nav bar iirc). Is this correct?

**Comment by alloy** [comment]
Ah, I was unsure where to add them and had to resist the urge to sort them alphabetically, so ended up appending.

**Comment by alloy** [comment]
@joeyAghion Ok, I added a sort script because I don’t want to think about this any further nor have others need to do so 😎

**Comment by joeyAghion** [comment]
Sorting alphabetically _definitely_ won't work. The visualizer will order the concentric rings based on the order in which those labels appear, as I found out here:



(I'm sorry for the trouble but hopefully my headaches can save you some. I tried to describe this in the README but it could use some clarification.)

Personally, I'd remove the script and implicitly encourage the right ordering by keeping the file consistent. If you like having the script, it just needs to ensure that "adopt," "trial," "assess," and "hold" appear in that order.

File: playbooks/technology_radar/data.csv (hunk 1 above)
Comment (damassi) [reply playbooks/technology_radar/data.csv]: @alloy - I'm of the opinion that for anything app-like that contains significant amounts of state we should continue using Redux (Prediction, Position). It's patterns scale very well, the ecosystem is vast, and so much comes out of the box for free (such as performance tuning, middleware pipelines, devtools, etc). There's some really nice hooks-centric, typescript-friendly Redux libs out there, such as [easy-peasy](https://github.com/ctrlplusb/easy-peasy). 

For Most Artsy webpage-like code (think, Force "app shell", new nav bar, etc) hooks + context API is more than enough.

Comment (alloy) [reply playbooks/technology_radar/data.csv]: That does seem to kinda follow my outline. I chose ‘asses’ to mean “do not use it by default, but if we determine that an app definitely benefits from a fuller redux-like env, only then do we use it”. But perhaps that should be ‘adopt’ and it’s implicitly clear that even for tech in ‘adopt’ we need to evaluate well on a case by case basis? (cc @joeyAghion)

**Comment by alloy** [comment]
Well that’s a little disappointing. I’m not sure I understand why their tooling doesn’t just sort things the intuitive way… 🤷‍♂ However, I don’t think we need to think about it any further, so I updated the script.

File: playbooks/technology_radar/data.csv (hunk 1 above)
Comment (damassi) [reply playbooks/technology_radar/data.csv]: I think 
> Redux,assess,languages & frameworks,FALSE,

Is accurate 👍

Comment (alloy) [reply playbooks/technology_radar/data.csv]: Done 👍

**Pull request merged**